src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/ReviewScheduleEtlController.kt

```diff
@@ -144,20 +140,6 @@ class ReviewScheduleEtlController(
     }
   }
 
-  private fun toSentenceType(legalStatus: LegalStatus): SentenceType =
```

Comment:
Because I moved this method out into the new mapper class (see below) it gave me an opportunity to make a few tidy ups in this class generally 👍